src/compas_timber/ghpython/components/CT_DrillHole/code.py

```diff
@@ -0,0 +1,32 @@
+from ghpythonlib.componentbase import executingcomponent as component
+from Grasshopper.Kernel.GH_RuntimeMessageLevel import Warning
+
+from compas_rhino.conversions import line_to_compas
+
+from compas_timber.ghpython import FeatureDefinition
+from compas_timber.parts import DrillFeature
+
+
+class DrillHoleFeature(component):
+    def RunScript(self, Beam, Line, Diameter, Length):
```

Comment:
is it required that the param names are uppercased? I think it does a positional match, so, these could be lowercased